sdk/proto/events/event.proto

```diff
@@ -66,6 +66,10 @@ message ActivityEvent {
 
 // Represents a security violation that is emitted by the agent
 message SecurityViolationEvent {
+  // DEPRECATED
```

Comment:
question: was the protobuf bindings regenerated and the vendor too after the change from `deprecated` to the `reserved` field approach that you took?

Reply:
Yes, when used `deprecated` that reflected only documentation wise. When used `reserved` the field is not in the message anymore, can't be used or set. So yep the generated struct for `SecurityViolationEvent` did change.

Reply:
Thanks, @mohamed-gougam